### DISCUSSION TITLE
project/web/capsule-render

### DISCUSSION THREAD

**Discussion by giscus** [kyechan99/kyechan99.github.io#41] [General]
# project/web/capsule-render

Dynamic Image

https://kyechan99.github.io/project/web/capsule-render

**Comment by rud1676** [comment]
이걸 누가 만들었지? 하고 봤는데 블로그글까지 너무 재미있게 보고가네요. 취준생인데 동기부여가 엄청됩니다. 궁금한게 요청이 많아지면서
Vercel로 배포하신 것에 무료버전은 55000건까지는 감당이 되는건가요? 앞으로 요청이 많아지고 Vercel로 한계가 생길 때 계획이 어떻게될지 궁금해서 여쭤봅니다!

**Reply by kyechan99** [reply]
안녕하세요!
블로그까지 따라와 주시고 감사드립니다. ㅎㅎ
취준도 화이팅입니다 !!

---

Capsule-Render 는 현재 (최근 한 달 기준) 약 11만건 가량 처리하고 있습니다!

![image](https://github.com/kyechan99/kyechan99.github.io/assets/14146566/8318ddec-a3d3-48c2-9567-6f113908184a)

Billing 에 대해서는, [Request](https://vercel.com/docs/limits/usage#requests) (요청량) 이 영향을 끼친다고는 하나 위와 같이 별도의 가격 [플랜](https://vercel.com/pricing)으로 구분되지는 않고 있습니다.
이는 위 프로젝트가 Serverless로 별도의 웹 서버 및 DB 등의 처리를  일절 사용하지 않고 요청 순간에 따른 JS 호출로 유지 및 운영되고 있는 면에서 이점을 보고 있지 않나 생각되네요.
한계가 오면 플랜을 당연 올릴 생각을 하고 있었습니다만 아직도 가야할 길이 머네요. 😄

**Comment by minari0v0** [comment]
개발자 취준생들 엄청 쓰고 있더라구요!!! 여기가 원조맛집이구나
저도 잘 쓰고 있습니닷 2026년도 빠이팅
